src/App.tsx

```diff
@@ -1,117 +1,16 @@
-import { useState, useEffect } from "react";
-import { Link, useNavigate } from "react-router-dom";
-import {
-  Box,
-  Button,
-  Container,
-  Flex,
-  Image,
-  ListItem,
-  UnorderedList,
-  Text,
-} from "@chakra-ui/react";
+import { Box, Container } from "@chakra-ui/react";
 import routes from "./routes";
-import { UserContext } from "./contextFile";
+import AuthProvider from "./context/auth";
+import Header from "./components/header";
 
 function App() {
-  const [auth, setAuth] = useState({ user: null });
```

Comment:
We can move all of this logic out of the `Application` component into a custom context.